lib/airbrake-ruby/async_sender.rb

```diff
@@ -41,6 +38,11 @@ def close
         raise Airbrake::Error, 'attempted to close already closed sender'
       end
 
+      unless @workers
```

Comment:
Covers the case when we create a notifier and close it immediately. If we do that, then `@workers == nil`, so we can return early and avoid failures about undefined methods.